chef/cookbooks/horizon/templates/suse/openstack-dashboard.conf.erb

```diff
@@ -54,6 +57,11 @@
         ExpiresActive on
         ExpiresDefault "access plus 1 month"
     </Location>
+    <% if @grafana_available %>
```

Comment:
I would do it the same way that is already done at line 33:
`<% unless @ssl_crt_chain_file.nil? or @ssl_crt_chain_file.empty? %>`